pkgs/games/endgame-singularity/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, unzip, python2Packages }:
+
+python2Packages.buildPythonApplication rec {
+  pname = "endgame-singularity";
+  version = "0.30c";
+  format = "other";
+
+  srcs =
+    [
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/singularity-${version}-src.tar.gz";
+        sha256 = "13zjhf67gmla67nkfpxb01rxs8j9n4hs0s4n9lnnq4zgb709yxgl";
+      })
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/endgame-singularity-music-007.zip";
+        sha256 = "0vf2qaf66jh56728pq1zbnw50yckjz6pf6c6qw6dl7vk60kkqnpb";
+      })
+    ];
+  sourceRoot = ".";
+
+  buildInputs = [ unzip ]; # The music is zipped
```

Comment:
This should be `nativeBuildInputs`. The difference is that `nativeBuildInputs` are used exclusively at build time. That is important for cross-compilation.

Similar to above comment, my style preference is one line per item for most lists. It makes for cleaner diffs when dependencies change and easier reordering.

Reply:
Also better for adding comments to individual items.